### PR TITLE
Adjust PPV rating algorithm

### PR DESCRIPTION
**Pull request by windymilla**
Instead of Some/Significant use Basic/Average/Complex Treat recording of illos the same as other features Require 3, not 4, complex features for project to be "Difficult" Round permitted number of errors to nearest integer, not downwards Reduce size-per-error values for Very Good rating by 20%

Also minor form changes:
Remove side bar
Remove spaces from br examples
Update the expected title format

### CODE REVIEW

**Comment by cpeel** [comment]
Thanks @windymilla! The dproofreaders codebases uses a prescribed coding style that, helpfully, can be applied automatically. See https://github.com/DistributedProofreaders/dproofreaders/blob/master/SETUP/CODE_STYLE.md for more details.

File: tools/post_proofers/ppv_report.php [hunk 1]
```diff
@@ -186,13 +186,30 @@ function maybe_report_form_problem($message)
...
+
         $problem = "";
+        $basic_id = "bsc_" . $feature_type;
+        $average_id = "avg_" . $feature_type;
+        $complex_id = "cpx_" . $feature_type;
+        $final_label = $feature_label_map[$feature_type];
         if ($action == HANDLE_ENTRY_FORM_SUBMISSION) {
-            if (isset($_POST[$some_id]) && isset($_POST[$sig_id])) {
-                $problem = _("You selected both \"Some\" and \"Significant Amount\".");
+            $numsel = 0;
```
Comment (cpeel) [inline tools/post_proofers/ppv_report.php:201]: What you have here is clear and I'm not suggesting that you change it, but the optimizer in me kept trying to figure out if there was a simpler way to do this and I think the answer is "no, but here's a more (untested) PHP way":
```php
if (count(array_intersect([$basic_id, $average_id, $complex_id], array_keys($_POST))) > 1)
```

**Review by cpeel — APPROVED** [review]
These updates look great to me - thank you for making them!

**Review by srjfoo — APPROVED** [review]

File: tools/post_proofers/ppv_report.php (hunk 1 above)
Comment (windymilla) [reply tools/post_proofers/ppv_report.php:201]: Yes, I can see how that would do the same - array_intersect would return all the keys from `$_POST` that match the basic, average or complex ids for this feature type. If there's more than one match, something's gone wrong?

That code looked less unwieldy when the `if`s were single line commands, but I fell foul of php-cs-fixer :)

Comment (cpeel) [reply tools/post_proofers/ppv_report.php:201]: I think my one-liner is probably harder to understand overall and think we should keep yours. I'll merge this in tomorrow morning my time unless you object between now and then 😁

**Comment by chrismiceli** [comment]
I wonder if we could enhance the existing unit tests to cover this?

File: tools/post_proofers/ppv_report.php (hunk 1 above)
Comment (windymilla) [reply tools/post_proofers/ppv_report.php:201]: I'm fine with that, but thanks anyway for the teaching point

**Comment by windymilla** [comment]
> I wonder if we could enhance the existing unit tests to cover this?

I've no idea what that involves. I would say this script is a relatively peripheral aspect of the site and doesn't lead to anything that is stored onsite or that can't be adjusted manually. So, it's probably not worth it, if it's a big effort.

**Pull request merged**